content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx

```diff
@@ -170,10 +170,13 @@ For the full range of supported operations, see the [MySQL full text search docu
 
 ### PostgreSQL
 
-To speed up your full-text queries, you should add an index to your database. Prisma Migrate currently doesn't support adding search indices in PostgreSQL, so this should be added in SQL. For example, the following SQL statement would add an index called `post_body_index` on the `body` column of the `posts` table:
+To speed up your full-text queries, you should add an index to your database. Prisma Schema currently doesn't support defining and Prisma Migrate doesn't support creating search indices, but you can easily add one yourself. The following example adds a `GIN` index to the table `posts` on the field `body`:
```

Comment:
```suggestion
To speed up your full-text queries, you should add an index to your database. Prisma schema and Prisma Migrate currently don't support defining and creating PostgreSQL search indices, but you can add one yourself. The following example adds a `GIN` index to the table `posts` on the field `body`:
```
I hope the wording here is correct @janpio

Reply:
Yes, the differentiation between "defining" and "creating" is probably not important enough to make the sentence that much harder to read than what I had.